### ISSUE TITLE
Can you add a download button?

### ISSUE THREAD

**Issue by Ketchup1138** [avluis/Hentoid#571]
Holding my finger on the picture to download a hentai take a lot of time. Can you add a download button to pictures of the manga? Right up corner?

Sorry for bad English.

**Comment by HaraDesu** [comment]
what do you mean, there are the download button in the toolbar below when you open up the manga series.

**Comment by Ketchup1138** [comment]
Yes I know but I'd like a button without open manga. I read manga after download.

**Comment by RobbWatershed** [comment]
@Ketchup1138 I'm not certain what you have in mind is feasible technically
That requires aligning an Android control (button) on HTML content (the cover)

**Comment by Ketchup1138** [comment]
I can wait I just wanted to say it and thanks.
